test/policy-collection/policy_generator_acm_hardening.go

```diff
@@ -0,0 +1,289 @@
+// Copyright Contributors to the Open Cluster Management project
+
+package e2e
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"math/rand"
+	"os"
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/types"
+
+	"github.com/stolostron/governance-policy-framework/test/common"
+)
+
+// generateInsecurePassword is a random password generator from 15-30 bytes. It is insecure
+// since the characters are limited to just hex values (i.e. 1-9,a-f) from the random bytes. An
+// error is returned if the random bytes cannot be read.
+func generateInsecurePassword() (string, error) {
```

Comment:
Could you please make this a public function in `test/common` to avoid duplicating this from the other Policy Generator test?

Reply:
yes, thanks for this comment.  I have moved the function.